include/cantera/oneD/IonFlow.h

```diff
@@ -67,12 +67,37 @@ class IonFlow : public StFlow
                               vector<double>& mobi_e);
 
 protected:
-    /*!
-     * This function overloads the original function. The residual function
-     * of electric field is added.
+
+    /**
+     * @brief Evaluate the electric field equation residual by Gauss's law.
```

Comment:
In Cantera, we use doxygen settings that make `@brief` and `@detail` directives unnecessary (same goes for all other instances).
```suggestion
     * Evaluate the electric field equation residual by Gauss's law.
```